src/libraries/System.Private.Xml/tests/TrimmingTests/XmlSerializer.Deserialize.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Xml.Schema;
+
+namespace System.Xml.Serialization.TrimmingTests
+{
+    internal class Program
+    {
+        // Preserve these types until XmlSerializer is fully trim-safe.
+        // see https://github.com/dotnet/runtime/issues/44768
+        [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(Response))]
```

Comment:
I'm surely missing something obvious but is there a reason why not annotating these other ones on the same PR? Specially since the idea is to get Serialize to be trim safe

Reply:
> is there a reason why not annotating these other ones on the same PR?

Because it is super expensive and not worth the amount of work at this time. See https://github.com/dotnet/runtime/issues/44768

> Specially since the idea is to get Serialize to be trim safe

The idea isn't to be trim safe for a fully trimmed app. The idea is for it to work in a Xamarin application, which doesn't trim user code. We do the same thing for the DataContractSerializer tests:

https://github.com/dotnet/runtime/blob/223fd3ac34ea7cb77c445ee32c7fdfd2a0dd8caa/src/libraries/System.Runtime.Serialization.Xml/tests/TrimmingTests/EndToEndTest.cs#L13-L14